### PR TITLE
Change fields into properties

### PR DESCRIPTION
**Pull request by Christoph142**
e.g. enables data binding

### CODE REVIEW

File: GeoAPI/License.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ GeoAPI is a set of .NET contracts and classes written in C# that
 define the fundamental operations required to validate a given
 geo-spatial data set to a known topological specification.
 
-Copyright (C) 2005-2014 NetTopologySuite team
+Copyright (C) 2005-2014 NetTopologySuite team, 2016 Airbus Defence and Space
```
Comment (DGuidi) [inline GeoAPI/License.txt]: what about this?

**Comment by DGuidi** [comment]
> enables data binding

as far as I know, to bind directly a class like `Coordinate` you need at least to handle `INotifyPropertyChanged`, something that is beyond the scope of `Coordinate` class itself. 
So I'm not sure about what kind of improvement this change can offer, although **_I suppose_** that there are no performance penalties ...

**Comment by kjeremy** [comment]
You only need to handle `INotifyPropertyChanged` if you want two-way data
binding.  This would at least give us one-way.  Any performance impact
should be negligible.

File: GeoAPI/License.txt (hunk 1 above)
Comment (Christoph142) [inline GeoAPI/License.txt]: I was told by our legal department that I need to add our company to the license so our modifications are covered by the LGPL you provided.

**Comment by Christoph142** [comment]
Yes, @kjeremy is right about this. I needed this change to enable databinding on a control that displays the hovered coordinate on a map. ;)

**Comment by DGuidi** [comment]
> I needed this change to enable databinding on a control that displays the hovered coordinate on a map

I suggest you to create a `CoordinateViewModel `class handle the PropertyChanged stuff

File: GeoAPI/License.txt (hunk 1 above)
Comment (DGuidi) [inline GeoAPI/License.txt]: > our modifications are covered by the LGPL you provided

I'm not a lawyer but as far as I know you can redistibute your own copy with your own modifications.
I'm pretty sure that LGPL doesn't force to change original NTS license file.

**Pull request closed without merging**